### DISCUSSION TITLE
"Skip refinement" option?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#65] [Q&A] [answered]
What is the meaning of this option? If I understand correctly there are two different steps: 1. "Reference-based pose estimation directly from 2D projections" and 2. "Fully constrained refinement of particle poses".  What step(s) are active, when "skip refinement" option is selected?

**Comment by abartesaghi** [comment]
nextPYP does only one kind of refinement, and that is directly from 2D projections. The main difference between the steps you mention is the range of the search (global vs. local) and other optimization parameters. "Skip refinement" should always be checked unless you want to refine the 2D projections independently (i.e., ala single-particle analysis, without the constraints), which in most cases would lead to overfitting and worsen map quality.

**Comment by JuhaHuiskonen** [comment]
How is global refinement switched on / off?  Is this purely by specifying a full range in Euler angles under "Constrained refinement" tab or how?  What angular ranges did you use for the ribosome test case in your paper?

**Reply by abartesaghi** [reply, marked as answer]
The range is part of it, the other part is the number of points you want to sample within the specified range, which can be controlled using the parameter `Optimizer - Exhaustive search points` in the `Constrained refinement` tab. You can look at the [classification tutorial](https://nextpyp.app/files/pyp/latest/docs/tutorials/tomo_empiar_10304.html#) where you will find the exact set of parameters we used to process EMPIAR-10304.
